types/externs.d.ts

```diff
@@ -372,8 +372,21 @@ declare global {
   }
 
   interface Window {
+    // Cached native functions/objects for use in case the page overwrites them.
+    // See: `driver.cacheNatives`.
+    __nativePromise: PromiseConstructor;
+    __nativeURL: URL;
+    __ElementMatches: Element['matches'];
+    __perfNow: Performance['now'];
+
+    /** Used for monitoring long tasks in the test page. */
+    ____lastLongTask?: number;
+
     /** Used by FullPageScreenshot gatherer. */
-    __lighthouseNodesDontTouchOrAllVarianceGoesAway: Map<HTMLElement, string>;
+    __lighthouseNodesDontTouchOrAllVarianceGoesAway: Map<Element, string>;
```

Comment:
hey wait a second how did this name end up staying like this 😆 I thought we had like 3 👎 comments

Reply:
> hey wait a second how did this name end up staying like this 😆 I thought we had like 3 👎 comments

ha, yes, I would have also preferred a different name there :)

Reply:
lol I only recall @brendankenny downvoting. seems I snuck some personality in the code without any approval, sry.

to be clear this is referencing React's [__SECRET_DOM_DO_NOT_USE_OR_YOU_WILL_BE_FIRED](https://www.reddit.com/r/javascript/comments/3m6wyu/found_this_line_in_the_react_codebase_made_me/cvcyo4a/).

Reply:
Haha I loved the concept I just didn't understand why it was backwards!! I'm so onboard for `__lighthouseNodesDontTouchOrAllVarianceGoesThroughTheRoof` :)

Reply:
I suppose I'm hoping someone finds it and suggests removing it to fix our variance problem..